KamerConnect/Models/Chat.cs

```diff
@@ -3,12 +3,15 @@
 public class Chat
 {
     public Guid ChatId { get; set; }
-    public Guid matchId { get; set; }
+    public Guid? matchId { get; set; }
     public List<ChatMessage> messages { get; set; }
```

Comment:
Hoe worden de messages nu gezet?

Reply:
Ow ja vergeten die functie van Sam aan te roepen in mijn functie, heb het toegevoegd!